server/sonar-server/src/main/java/org/sonar/server/measure/index/ProjectMeasuresIndex.java

```diff
@@ -222,16 +224,6 @@ private static AbstractAggregationBuilder createQualityGateFacet() {
     return qualityGateStatusFilter;
   }
 
-  private static AbstractAggregationBuilder createLanguagesFacet() {
-    return AggregationBuilders.nested("nested_" + FILTER_LANGUAGE)
-      .path(FIELD_LANGUAGES)
-      .subAggregation(
-        AggregationBuilders.terms(FILTER_LANGUAGE)
-          .field(FIELD_LANGUAGES_KEY)
-          .subAggregation(AggregationBuilders.sum("size_" + FILTER_LANGUAGE)
-            .field(FIELD_LANGUAGES_VALUE)));
-  }
-
   private static AbstractAggregationBuilder createTagsFacet() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createTagsFacet" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)